pkg/storage/export/export/export_test.go

```diff
@@ -989,7 +1068,7 @@ func createVMVMExport() *exportv1.VirtualMachineExport {
 				Kind:     "VirtualMachine",
 				Name:     testVmName,
 			},
-			TokenSecretRef: "token",
+			TokenSecretRef: &token,
```

Comment:
What you did is not wrong, this is just an alternative:
```go
 TokenSecretRef: pointer.StringPtr("token")
```
Saves you from having to create the variable at the top.